StandaloneMmPkg/Drivers/StandaloneMmIplPei/MmFoundationHob.c

```diff
@@ -803,6 +803,49 @@ GetRemainingHobSize (
   }
 }
 
+/**
+  Check if Mm Core FV HOB was built.
+
+  Check if Mm Core FV HOB was built on platform HOB list,
+  if yes, skip building MM Core FV HOB,
+  if No, continue to build MM Core FV HOB
+
+  @param[in]      PlatformHobList     Platform HOB list.
+  @param[in]      PlatformHobSize     Platform HOB size.
+
+  @retval TRUE    Skip building MM Core FV HOB.
+          FALSE   Continue to build MM Core FV HOB.
+**/
+BOOLEAN
+IsMmCoreFvHobBuilt (

```

Comment:
why not merge this PR into the https://github.com/tianocore/edk2/pull/6547 as one of commit?